packages/tokens/src/global/colors.ts

```diff
@@ -291,12 +291,12 @@ export const marigold: ColorVariants = {
 };
 
 export const yellow: ColorVariants = {
-  primary: '#fde300',
-  shade10: '#e4cc00',
-  shade20: '#c0ad00',
-  shade30: '#817400',
-  shade40: '#4c4400',
   shade50: '#282400',
```

Comment:
There is no change to color tokens here. Just reordered to match token pipeline output.